crates/bevy_pbr/src/render/light.rs

```diff
@@ -176,6 +178,7 @@ pub fn extract_lights(
     global_point_lights: Extract<Res<GlobalVisibleClusterableObjects>>,
     point_lights: Extract<
         Query<(
+            &RenderEntity,
```

Comment:
I don't have the brainpower to review this file atm. Requesting that one of the other reviewers please carefully review this one as I did not.